### ISSUE TITLE
Support `ember-modifier@4`

### ISSUE THREAD

**Issue by sebastianhelbig** [cibernox/ember-basic-dropdown#689]
Currently when trying to use it inside a project which uses `ember-modifier@4.0.0`, compilation is broken:

```
ember-basic-dropdown needs to depend on ember-auto-import in order to use ember-modifier
```

**Comment by jelhan** [comment]
We need to do the following bare minimum to land `ember-modifier@^4` support:

- [ ] Widen version range for `ember-modifier` to include `^4.0.0`
- [ ] Add `ember-auto-import@^2` as a dependency (breaking change)
- [ ] Drop support for node 12 (breaking change)

I may have some time later this week to provide PRs for each of them. Happy to see that someone else was quicker. :laughing:

**Comment by mkszepp** [comment]
@cibernox this is fixed with #696 and rollout was in v7, so we can close it
